### ISSUE TITLE
Voucher amount is not decreasing

### ISSUE THREAD

**Issue by nzimelimited** [verbb/gift-voucher#82]
We are using your craft-3 dev branch on a site currently in development because of issues in 2.6.3 which seem to have been resolved in the dev branch.

Since updating to the last commit our voucher amounts are not decreasing after submitting an order and the order is not being linked to the voucher code. Everything else is working perfectly, correct tax, shipping, totals, etc which are the issues we've previously had.

Looking at the code, line 89 of `src/services/CodesService.php` is checking the order sourceSnapshot field for giftVoucherPluginCode to determine if the adjuster line is a voucher. However this param is not present in this object for our orders.

**Additional info**
Plugin version: dev-craft-3
Craft version: 3.7.16
Multi-site: Yes

**Comment by engram-design** [comment]
Good catch, I'd not reverted that part yet, as there's been a lot of back-and-forth with the adjuster changes.

Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-3 as 2.6.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.7.0](https://github.com/verbb/gift-voucher/releases/tag/2.7.0)
